package.json

```diff
@@ -22,7 +22,7 @@
     "react-component"
   ],
   "peerDependencies": {
-    "react": ">=0.13.0"
+    "react": ">=0.13.0 <0.15.0 || ^0.14.0-alpha"
```

Comment:
@namuol Hey Louise, @jpuri and I tested it quite intense. Good job. We noticed the 0.14 is already at beta3 and only tested it for this version.

Our idea was to make the version explicit to make sure people don't run into trouble: `"react": ">=0.13.0 || ^0.14.0-beta3"`. Once beta4 is out we can migrate to it and verify there are no breaking changes. Are you using the `0.14.0-alpha` anywhere or is it fine like that?